front-end-master/about_us/about.js

```diff
@@ -45,3 +45,16 @@ class Person{
 		 this.DOMnode.appendChild(bioElem);
      }
 }
+
+var div_select_arr = document.getElementsByClassName('div-name');
+
+function gg(element){
```

Comment:
I'm assuming these functions are for debugging purposes? If don't need them anymore, remove them. Otherwise, change the names to reflect what they do.

Reply:
sorry. my bad.